src/opex_dashboard/tags/uri_to_regex.py

```diff
@@ -12,4 +12,4 @@ def uri_to_regex(value: object) -> str:
     Returns:
         str -> The serialized object
     """
-    return re.sub("{[^/]+}", "[^/]+", str(value)) + "$"
+    return re.sub("{[^/]+}", "[^/]+", str(value)) + "($|?)"
```

Comment:
I've tested on one of our dashboard, and it seems this change will introduce a regression on generated query (the one  that we fixed in https://github.com/pagopa/opex-dashboard/pull/43)